x-pack/solutions/observability/plugins/observability/server/lib/rules/custom_threshold/types.ts

```diff
@@ -54,6 +55,7 @@ export type CustomThresholdAlertState = AlertState; // no specific instance stat
 export type CustomThresholdAlertContext = AlertContext & {
   alertDetailsUrl: string;
   group?: object;
+  grouping?: object;
```

Comment:
can we change it to `Record<string, any>`?

Reply:
I'll change it.